API_CHANGELOG.md

```diff
@@ -90,16 +90,16 @@ This is obsolete. Use getRegion('name') instead.
 * TemporalMemory `compute()` and `activateCells()` now use only SDR variants, old overloads with C-style arrays removed. Bindings and 
   tests also updated. 
   
-* Changed all use of nupic to htm.   This means that Python users must import from 
- - htm.bindings.algorithms
- - htm.bindings.engine_internal
- - htm.bindings.math
- - htm.bindings.encoders
- - htm.bindings.sdr
+* Changed all use of nupic to htm.   This means that C++ users must include from
+    - <htm/algorithms/*.hpp>
+    - <htm/engine/*.hpp>
+    - <htm/math/*.hpp>
+    - <htm/encoders/*.hpp>
+    - <htm/types/*.hpp>
```

Comment:
this is ok, another option is to keep the log chronological, so the next entry would be "renamed all of NuPIC instances to HTM (as in namespaces, ..)"